sdk/search/Azure.Search.Documents/Directory.Build.props

```diff
@@ -33,6 +36,13 @@
       Include="Azure.Core.Experimental" />
   </ItemGroup>
 
+  <!-- Other package references -->
+  <ItemGroup>
+    <PackageReference
+      Condition="'$(UseAzureCoreExperimentalSpatial)' != 'true'"
+      Include="Microsoft.Spatial" />
```

Comment:
We can't ship with a dependency on Microsoft.Spatial.  It should have its own feature flag so we don't need to `'$(UseAzureCoreExperimentalSpatial)' != 'true'`.

Reply:
I will have to cut Microsoft.Spatial entirely, then. You had me add this to the test project when `FieldBuilder` was a sample to make sure it still works; however, with moving `FieldBuilder` to the shipping project, it came along. I agree we shouldn't ship it as a dependency, but see no other choice than to remove spatial support.

Reply:
We're removing Spatial support elsewhere too for the same reason (i.e., SearchDocument, SearchFilter, etc.) until we can take a dependency on whatever ships in Azure.Core.  It's fine to leave with a new flag if you want it for testing, etc., but it should be off for the next release.